config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-ci.yaml

```diff
@@ -22,3 +22,11 @@ periodics:
         # we need privileged mode in order to do docker in docker
         securityContext:
           privileged: true
+        volumeMounts:
+          - name: host-dev
+            mountPath: /dev
+      volumes:
+      - name: host-dev
+        hostPath:
+          path: /dev
```

Comment:
Whoa, I don't think we should be mounting all of /dev rw 😮

Reply:
@figo it might be enough to just mount `/dev/loop0` and `/dev/loop1`

Reply:
@fejta @BenTheElder i am limiting the scope to /dev/loop0 and /dev/loop1, is this acceptable for prow in short term? i would like to work on long term solution (avoid the host dev) with @frapposelli

Reply:
What is the issue where we can track progress on this long term solution? Will you add the URL to that issue in this volume?

Reply:
This will be resolved as part of the larger "Image stamping" effort in Cluster API, ref: kubernetes-sigs/cluster-api#723